java/play-a-kml-tour/README.md

```diff
@@ -2,75 +2,45 @@
 
 Play tours in KML files.
 
-![Play a KML tour App](play-a-kml-tour.png)
+![Image of play KML tour](play-a-kml-tour.png)
 
 ## Use case
 
-KML, the file format used by Google Earth, supports creating tours,
-which can: control the viewpoint of the scene, hide and show content,
-and play audio. Tours allow you to easily share tours of geographic
-locations, which can be augmented with rich multimedia. Runtime allows
-you to consume these tours using a simple API.
+KML, the file format used by Google Earth, supports creating tours, which can control the viewpoint of the scene, hide and show content, and play audio. Tours allow you to easily share tours of geographic locations, which can be augmented with rich multimedia. Runtime allows you to consume these tours using a simple API.
 
 ## How to use the sample
 
-The sample will load a KMZ file. When a tour is found, the buttons will
-be enabled. Use _Play_ and _Pause_ to control the tour. When you're
-ready to show the tour, use _Reset_ to return the tour to the unplayed
-state.
+The sample will load the KMZ file from ArcGIS Online. When a tour is found, the _Play_ button will be enabled. Use _Play_ and _Pause_ to control the tour. When you're ready to show the tour, use the reset button to return the tour to the unplayed state.
 
 ## How it works
 
-1. Load the KML file and add it to a layer. 
-2. Create the KML tour controller. Wire up the buttons to the `Play`,
-   `Pause`, and `Reset` methods. 
-1. Explore the tree of KML content and find
-   the first KML tour. Once a tour is found, provide it to the KML tour
-   controller. 
-4. Enable the buttons to allow the user to play, pause, and reset the
-   tour.
-   
+1. Create `KmlDataSet` from the local kmz file and instantiate a layer from it with `new KmlLayer(kmlDataSet)`.  
+2. Create the KML tour controller. Wire up the buttons to the `kmlController.play()`, `kmlController.pause()`, and `kmlController.reset()` methods.
+3. Explore the tree of KML content to find the first KML tour. Once a tour is found, provide it to the KML tour controller.
+4. Enable the buttons to allow the user to play, pause, and reset the tour.
```

Comment:
This step seems redundant, after it's already said to wire up the buttons in step 2.